app/models/vault/tag.rb

```diff
@@ -32,5 +32,28 @@ def Tag::tags_list(pid)
       ).group('vault_tags.name').group('vault_tags.id').map(&:name) #OPTIMIZE_ME!
     end
 
+    def Tag::color(tag)
```

Comment:
Could you make colours as enum in separate file, please? It will make changing colours more obvious. If won't do it, still accept merge request.

Reply:
testing code in branch [release_0.5.0](https://github.com/noshutdown-ru/vault/tree/release_0.5.0)
will move it